src/main/java/frc/robot/constants/DriveConstants.java

```diff
@@ -13,10 +13,8 @@
 public class DriveConstants {
 
   // TODO 1.1.1: Set these
-  public static final int LEFT_MOTOR_1_ID = -1;
-  public static final int LEFT_MOTOR_2_ID = -1;
-  public static final int RIGHT_MOTOR_1_ID = -1;
-  public static final int RIGHT_MOTOR_2_ID = -1;
+  public static final int LEFT_MOTOR_1_ID = 11;
+  public static final int RIGHT_MOTOR_1_ID = 6;
```

Comment:
Deleting these lines causes an error. Add them back or remove the reference causing the error.